roles/cifmw_external_dns/tasks/requirements.yml

```diff
@@ -0,0 +1,86 @@
+---
+# Copyright 2024 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Ensure we have cifmw_openshift_kubeconfig
+  delegate_to: localhost
```

Comment:
Care to provide some context? why `delegate_to: localhost` ? where is it running and what's localhost in this context?
My understanding: this will be called from the "hook" in the automation, hence it'd be running from "controller-0" against "compute/ceph" - so "localhost" is he controller-0. Correct?

If so, care to see if we can have a `run_once: true` ? Else it would run the same, over and over, for each compute/ceph node.

Reply:
Yes, localhost is controller-0

I will add a `run_once: true` in addition to delegation.